### PR TITLE
Chef 16 Fixes

### PR DESCRIPTION
**Pull request by zimmerry**

### CODE REVIEW

**Comment by zimmerry** [comment]
[Chef 16: osl-docker](https://trello.com/c/3AEfyxBi/2008-chef-16-osl-docker)

**Comment by zimmerry** [comment]
Chef 15 wants quotes around the numbers for cron, Chef 16 doesn't.

File: recipes/nvidia.rb [hunk 1]
```diff
@@ -119,7 +119,11 @@
 log 'yum makecache cuda' do
   message 'yum makecache cuda'
   only_if { ::File.exist?(makecache_file) }
+end
```
Comment (ramereth) [inline recipes/nvidia.rb:122]: Shouldn't this log resource go away and be replaced by the notify_group resource below?

**Review by ramereth — CHANGES_REQUESTED** [review]

**Comment by ramereth** [comment]
> Chef 15 wants quotes around the numbers for cron, Chef 16 doesn't.

What happens if you explicitly set the parameters as strings in the resources?

File: recipes/nvidia.rb (hunk 1 above)
Comment (zimmerry) [reply recipes/nvidia.rb:122]: I kept it because it was printing a message to the log. Do you want me to remove it?

Comment (ramereth) [reply recipes/nvidia.rb:122]: Yes please.

**Review by ramereth — APPROVED** [review]
Please do a minor bump.

**Comment by zimmerry** [comment]
!bump minor ~

**Pull request merged**